### ISSUE TITLE
autosave current file before running test

### ISSUE THREAD

**Issue by rogercampos** [skalnik/vim-vroom#44]
It would be nice if the current file is autosaved before running tests either with `,r` or `,R`.

**Comment by rschenk** [comment]
Agreed. Or more specifically, on my machine `,r` and `,R` autosave, but annoyingly `,l` does not, which is the one I use most frequently
